src/containers/Login/Login.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Formik, Field, Form, FormikHelpers } from 'formik';
+
+import { styled } from '../../styles';
+import { Button, InputField, PasswordInput } from "../../components";
+import loginLandscape from '../../img/loginLandscape.jpg';
```

Comment:
rename the img folder to 'assets'
then create img folder there with all images
use index file there